app/javascript/test/SearchResultsLayout.test.tsx

```diff
@@ -1,7 +1,52 @@
 import '@testing-library/jest-dom';
-import { render, screen } from '@testing-library/react';
-import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import React from 'react';
 import SearchResultsLayout from '../components/SearchResultsLayout';
+import {Header} from '../components/Header';
+
+describe('Header', () => {
```

Comment:
I think we should move `Header` tests to a new file.

Reply:
Sure - sounds good.